test/integration/tasks.test.js

```diff
@@ -820,4 +820,38 @@ describe("Tasks", function () {
       expect(res.body.message).to.be.equal("No overdue tasks found");
     });
   });
+
+  describe("PATCH /tasks/updateOldTaskStatus/all", function () {
+    it("Should successfully update old tasks with 'DONE' and 'UNASSIGNED' statuses when authenticated as SUPERUSER", function (done) {
+      chai
+        .request(app)
+        .patch("/tasks/updateOldTaskStatus/all")
+        .set("cookie", `${cookieName}=${superUserJwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Updated Old tasks");
+          expect(res.body.tasks).to.be.a("array");
```

Comment:
asserting type to be an array doesn't assures much we should also check the content in the array.